pages/Messages/Overview.md

```diff
@@ -175,83 +99,22 @@ A visual reply to a post.
-
-| dsnpData field | description | type | bloom |
-| ------------- |------------- | ---- | --- |
-| fromAddress | id of the sender | bytes20  | YES
-| data | encrypted graph change data | string | no
-| contentHash | keccak-256 hash of unencrypted content | bytes32 | no
-
-#### PrivateBroadcast
-
-An encrypted Broadcast decipherable by specific accounts.
-This describes the format once decrypted.
-
-| dsnpData field | description | type | bloom |
-| ------------- |------------- | ---- | --- |
-| fromAddress | id of the sender | bytes20 | YES
-| inReplyTo | ID of the message the broadcast references |  bytes32
-| contentHash      | keccak-256 hash of content stored at URI |  bytes32 | no
-| uri       | content uri | string | no
-
-
-### Unified Announcement Format
-
-This is currently not the recommended solution, but is presented as a comparison.
-
-* All actions are posted to the chain with some pertinent information about the action
-* The same information is posted regardless of action
-* The rest of the information for the action is stored off chain.
-* **Advantages**
-    * less data on chain --> lower up-front costs and lower node storage requirements
-    * privacy management is easier -- for example, "right to be forgotten" is easier to comply with since
-        data is external, off-chain.
-* **Disadvantages**
-    * indexing requires retrieving content @ uri
-    * archiving requires retrieving more content @ uri
-    * validation requires retrieving more content @ uri
-    * cost savings may be minimal
-
-| field | description | type
-|-------|-------------| ---|
-| topic | Ethereum log topic |  bytes
-| action type | the type of action | bytes
-| fromAddress | social identity | bytes
-| uri | uri of stored action information | string
-
-### All data on chain
-
-One possibility is not to have any data stored off-chain; instead, even the ActivityPub content would be posted to chain.
-The disadvantages far outweigh the advantages:
-
-* **Advantages**
-    * Validation, indexing, discovery are much easier
-* **Disadvantages**
-    * This amount of data would rapidly slow down the network
-    * The posting of illegal content could potentially shut down the network
-    * Garbage collection, validation, privacy concerns, and dealing with illegal content become interdependent, and would pose conflicting interests.
-    * Would still need archive to store at least some content
-    * Unknown, likely chilling effect on incentive models
-
-### No data except hashes on chain
-
-Only hashes of the events are stored on chain; everything else is interpreted via API(s)
+#### Other notes
+1. Only some Liberty Platform activities will have the full message posted to chain. Examples:
+    * Public messages (profile changes, public posts, reactions)
+    * Direct messages between accounts (dead drop, inbox)
+    * Key changes (private graph KeyList, encryption KeyList, public key rotation)
+    * Graph changes (follow/unfollow)
```

Comment:
I think we can drop this section. I think it is more confusing than leaving it out. Also the current list of examples appears to just be everything